### DISCUSSION TITLE
Change Champion Wallace Dialog

### DISCUSSION THREAD

**Discussion by Tyrranux** [haven1433/HexManiacAdvance#225] [Q&A]
It seemed to let me edit the Elite Four just fine. But then I got to the room with Wallace in it, nothing. I click on the NPC sprite, "no script". My entire project hinges on being able to change the dialog of the whole game, yet it seems I can't even edit the script here, any attempts to try just brings me back to the previous room. What do I do?! Do I just scour the ENTIRE hexadecimal table for hours or days going through the code with a comb?! Where is the option to edit Wallace's dialog? Please help and thank you.

![CLIP](https://github.com/haven1433/HexManiacAdvance/assets/169882680/07fc469f-9041-4059-adcc-8bfe9bfeb7d2)

**Comment by haven1433** [comment]
![image](https://github.com/haven1433/HexManiacAdvance/assets/10834600/e02330bf-9e08-481f-8646-5b103204ee6c)
That map uses a "pre-frame" script to control the encounter between the player and Wallace. You can edit the script in the map's header.

Questions like these are best answered on the [Community Discord](https://discord.com/invite/x9eQuBg), where lots of active users can help discuss issues and find solutions together.
